WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -207,21 +197,13 @@ import WordPressShared
                 .Published: published]
     }
     
-    func noResultsMessageText() -> String {
-        if syncHelper?.isSyncing == true || isSearching() {
+    override func noResultsMessageText() -> String {
+        if syncHelper.isSyncing == true || isSearching() {
             return ""
         }
         
         let filter = currentPostListFilter()
-        
-        // currentPostListFilter() may return `nil` at this time (ie: it's been declared as
-        // `nullable`).  This will probably change once we can migrate
-        // AbstractPostListViewController to Swift, but for the time being we're defining a default
-        // filter here.
-        //
-        // Diego Rey Mendez - 2016/04/18
-        //
-        let filterType = filter?.filterType ?? .Draft
+        let filterType = filter.filterType
```

Comment:
@diegoreymendez super nit-picky, but can we do `currentPostListFilter().filterType` here to match the same line in `PageListViewController.swift Line:146`